### PR TITLE
Add config.owner to v1 syntax for setting UID and GID

### PR DESCRIPTION
**Pull request by aseaday**
This is a extra pr for #410.
There three ways to set UID/GID during building:

1. read args from cli
2. config.owner in build.envd
3. get current user's uid/gid
and their priorities.

### CODE REVIEW

**Review by gaocegege — APPROVED** [review]
/cc @kemingy 

LGTM

File: pkg/lang/frontend/starlark/v1/config/config.go [hunk 1]
```diff
@@ -225,3 +226,24 @@ func ruleFuncRepo(thread *starlark.Thread, _ *starlark.Builtin,
...
+
+	if err := starlark.UnpackArgs(ruleOwner, args, kwargs, "uid", &envdUid, "gid?", &envdGid); err != nil {
+		return nil, err
+	}
+	uid, _ := envdUid.Int64()
+	gid, _ := envdGid.Int64()
+	if uid < 0 || uid > 65535 || gid < 0 || gid > 65535 {
+		err := errors.New("get a wrong uid or gid")
+		return nil, err
+	}
```
Comment (kemingy) [inline pkg/lang/frontend/starlark/v1/config/config.go:245]: If `gid` is optional, seems it will fail because `gid=-1` by default.

Comment (aseaday) [reply pkg/lang/frontend/starlark/v1/config/config.go:245]: I think I should modify the optional properity.

**Review by kemingy — APPROVED** [review]
LGTM.

Shall we support this feature in v0? @gaocegege

**Comment by gaocegege** [comment]
Let's merge it and see. I think there is no need to support for v1 now. WDYT

**Comment by aseaday** [comment]
I don't think we need to support it in v1. I think all v0 update from today should be only bug fix or critical issues.

**Pull request merged**